modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -1295,32 +1302,88 @@ GAPI_FLUID_KERNEL(GFluidMulCOld, cv::gapi::core::GMulCOld, true)
     }
 };
 
-GAPI_FLUID_KERNEL(GFluidDivC, cv::gapi::core::GDivC, false)
+GAPI_FLUID_KERNEL(GFluidDivC, cv::gapi::core::GDivC, true)
 {
     static const int Window = 1;
 
-    static void run(const View &src, const cv::Scalar &_scalar, double _scale, int /*dtype*/,
-                    Buffer &dst)
+    static void run(const View& src, const cv::Scalar& _scalar, double _scale, int /*dtype*/,
+                    Buffer& dst, Buffer& scratch)
```

Comment:
The run() function of each GAPI kernel must have a strictly defined API, otherwise a compilation error will occur. The `const` keyword placed before `Buffer& scratch` changes API of this function and can cause compilation failures.